test/access.jl

```diff
@@ -64,10 +65,11 @@
         @test iterate(RNAKmer("ACUG"), 4) == (RNA_G, 5)
         
 
-        @test iterate(RNAKmer("ACUG"), 1)  !== nothing
+        @test !isnothing(iterate(RNAKmer("ACUG"), 1))
```

Comment:
```suggestion
        @test iterate(RNAKmer("ACUG"), 1) !== nothing
```

don't want to just change the new one, my bad.